docs/source/checkpointing.rst

```diff
@@ -22,6 +22,24 @@ to pass through a single process.
 It also supports flexible checkpointing, where one can save meshes and
 :class:`~.Function` s on :math:`N` processes and later load them on
 :math:`P` processes.
+If :math:`P == N`, the parallel distribution and entity permutation
+(reordering) of the saved mesh is recovered on the loaded mesh by default.
+The distribution and permutation data are by default stored under names
+automatically generated by Firedrake, but it is recommended that users
```

Comment:
I think I don't understand this recommendation. Won't most users will be fine with the automatically generated names?

Reply:
Yes. Updated.